artview/components/plot_radar.py

```diff
@@ -479,13 +472,22 @@ def toolDefaultCmd(self):
 
     def _set_fig_ax(self):
         '''Set the figure and axis to plot.'''
-        if (self.airborne) or (self.rhi):
+        self.XSIZE = 8
+        self.YSIZE = 8
+        self.fig = Figure(figsize=(self.XSIZE, self.YSIZE))
```

Comment:
Glad you moved the above lines in here. Makes sense.
Amazing what a fresh pair of eyes can see.